benches/benchmarks.rs

```diff
@@ -143,44 +121,46 @@ fn encoding_test_data(row_cnt: usize, col_cnt: usize) -> Value {
 
 fn encoding_benchmarks(c: &mut Criterion) {
     let mut group = c.benchmark_group("Encoding");
-    let test_cnt_pairs = [(100, 5), (100, 15), (10000, 5), (10000, 15)];
-    for (row_cnt, col_cnt) in test_cnt_pairs.into_iter() {
-        for fmt in ["json", "msgpack"] {
-            group.bench_function(&format!("{fmt} encode {row_cnt} * {col_cnt}"), |b| {
-                let mut res = vec![];
-                let test_data = PluginOutput::CallResponse(
-                    0,
-                    PluginCallResponse::value(encoding_test_data(row_cnt, col_cnt)),
-                );
-                let encoder = EncodingType::try_from_bytes(fmt.as_bytes()).unwrap();
-                b.iter(|| encoder.encode(&test_data, &mut res))
-            });
-        }
+    let row_cnt = 10000;
```

Comment:
why are these row_cnt and col_cnt hard coded?

Reply:
oh wait, they were hard coded to begin with but there were several benchmarks, now there's just 1 with 10k and 15. Maybe we should at least keep 2, one small and one bigger?

Reply:
Previously we stored these in a vec and iterated over it, now I removed the other so its only 1 left. And I thought it fine to have it hard coded as its just used for the benchmark.

Reply:
Fixed, but i still think its quite a lot at least as most of the current bench suit is encoding and decoding messagepack and json.